pepper/test/e2e/hash_transform.sh

```diff
@@ -1,18 +1,14 @@
 #!/bin/bash
 set -e
+source ./test/e2e/common.sh
 
 cd $PEPPER/pepper
 
 sed -i 's/#define ORDERS [0-9]*/#define ORDERS 2/g' apps/hash_transform.h
 
-./pepper_compile_and_setup_P.sh hash_transform
-./pepper_compile_and_setup_V.sh hash_transform transform.vkey transform.pkey
+echo 182713746507528768497599437701574618975 > prover_verifier_shared/hash_transform.inputs
```

Comment:
I think it might be nice if we could make e2e-test with any number of orders (and maybe a default of 2). Could we possibly change the python script to write the hash_transform.inputs directly to this file?

Reply:
We can, we just have to change the `ORDERS` value in the `sed` command above (and update the echo value accordingly). We could change the python script to only output the SHA hash in the correct format (instead of SHA and Pedersen), but this feels unrelated to this PR. 
Please feel free to make the tests more convenient if you see a gap.